### PR TITLE
Add conference paper citation

### PR DESCRIPTION
**Pull request by RMeli**

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -54,3 +54,31 @@ keywords:
 license: BSD-3-Clause
 version: 0.5.0
 date-released: '2024-05-31'
+preferred-citation:
```
Comment (msimberg) [inline CITATION.cff:57]: Does `preferred-citation` mean that this is preferred over the "software DOI"? Do we have a preference to cite one or the other?

Comment (RMeli) [reply CITATION.cff:57]: It's the citation that will appear on GitHub under the "Cite this repository". Unfortunately I don't know of a way of adding the paper to `CITATION.cff` without using the `preferred-citation` field. (Ideally, both the software and the paper should be cited.)

Comment (rasolca) [reply CITATION.cff:57]: I would prefer to keep the software DOI as primary citation, as papers and posters can get outdated very soon.
We might add other references to the README

**Review by rasolca — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -43,6 +43,33 @@ target_link_libraries(<your_target> PRIVATE DLAF::DLAF)
...
+        and Bosilca, George",
+    title="DLA-Future: A Task-Based Linear Algebra Library Which Provides a GPU-Enabled Distributed Eigensolver",
+    booktitle="Asynchronous Many-Task Systems and Applications",
+    year="2024",
+    publisher="Springer Nature Switzerland",
+    address="Cham",
+    pages="135--141",
+    isbn="978-3-031-61763-8"
+}
+
```
Comment (msimberg) [inline README.md:70]: Nit:
```suggestion
```

**Review by msimberg — APPROVED** [review]

**Pull request merged**